automation/newsletter.py

```diff
@@ -3,7 +3,7 @@
 import pathlib
 from typing import Generator
 
-import frontmatter
```

Comment:
What's being ignored here? If an ignore is strictly necessary, please include an error code (such as `type: ignore[misc]`, replacing `misc` with whatever the specific code is).

Reply:
ignore was placed as per - https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-library-stubs-or-py-typed-marker

From what I understand there is no typing from the module so it doesn't have anything to check against.

Reply:
Yes, I'm asking for a specific error code as per https://mypy.readthedocs.io/en/stable/error_codes.html#displaying-error-codes. A blanket `type: ignore` is dangerous, IMHO, because it can mask additional issues (and I made a [flake8 plugin](https://pypi.org/project/flake8-type-ignore/) for this reason).

Reply:
Ah the error is import. I've added the more verbose ignore